content/working_with/official_plugins/Orchestration/terraform.md

```diff
@@ -122,7 +122,62 @@ This refers to a Terraform module.
 
     * flags_override: The plugin has its own internal logic for appending flags to the tflint command.  However, if you wish to add or modify flags, configure here.  For example, "{'loglevel': 'debug'}", becomes "--loglevel=debug".
     * env: Additional env vars for duration of tflint executions,
-
+    * enable: boolean, In order for it to work, must mark True.
+    
+  * tfsec_config:  tfsec is a static analysis security scanner for your Terraform code.
+    * installation_source: The URL to download the tfsec binary from, e.g. 'https://github.com/aquasecurity/tfsec/releases/download/v1.1.3/tfsec-linux-amd64'.
+    * executable_path:  If the binary is located on the file system, this is the path on the file system, e.g. /usr/local/bin/tfsec.
```

Comment:
maybe say "If the binary is already located on your system (you installed it manually), this is the path on the file system...."